### ISSUE TITLE
RGI16: Mt Kenya

### ISSUE THREAD

**Issue by rainerprinz** [GLIMS-RGI/rgi7_scripts#38]
This is a very picky comment as these glaciers are sooo small. This glacier I marked consists of two glaciers: Tyndall (the larger one) and Heim (the small part I marked). One glacier is missing: Northey. See Rostom & Hastenrath 2007: 10.3112/erdkunde.2007.03.05: https://www.erdkunde.uni-bonn.de/archive/2007/variations-of-mount-kenyas-glaciers-1993-2004

G037305E00152S

**Comment by fmaussion** [comment]
Thanks for the feedback @rainerprinz ! 

> This is a very picky comment as these glaciers are sooo small.

All feedback is welcome. Whether we implement it or not is a different story ;-)  

Is the outline available in GLIMS? See https://www.glims.org/maps/glims . If it is, then including it in RGI is no problem. If not, than someone needs to submit it to GLIMS so that it can be included in RGI. 

In any case, we will leave this issue open until resolution.

**Comment by iamdonovan** [comment]
I came across this issue while diligently avoiding some admin work earlier today, and managed to georeference the map from the paper referenced by @rainerprinz, as well as the [earlier paper](https://www.jstor.org/stable/25646591) by the same authors with outlines from 1993:

![Screenshot from 2023-01-31 20-40-54](https://user-images.githubusercontent.com/5552374/215878449-1dc4ddd4-463a-4330-8e6e-5e42482a5293.png)

From what I can tell from the maps, Diamond Glacier (labeled on both the 1993 and 2004 maps) is also missing from GLIMS, which only has outlines from 1993 (though not the date listed in the 1994 paper).

**Comment by fmaussion** [comment]
I don't know what to do with this information ;-) - are new outlines going to glims soon?

**Comment by rainerprinz** [comment]
Mapping these little glaciers is delcate. High resoultion images are preconditioned. Sometimes there is too much snow cover, somtimes they are barley visible because of shadows. Diamond glacier still exists, only ~200 m² though. I was hesitating submitting the MtK inventory, as we just prepare a new one for all East African glaciers (Mt Kenya, Rwenzori and Kilimanjaro). Paper will be submitted soon.

**Comment by iamdonovan** [comment]
@fmaussion - fair enough, I apparently forgot to add my question. I wasn't 100% sure about sending these to GLIMS, given that this is just digitized from maps created by someone else, and they should probably get the "credit"?

@rainerprinz  - do you already have versions of the 1993/2004 outlines, in addition to the updated East African inventory?

**Comment by rainerprinz** [comment]
I don't have the full outlines for 1993/2004. I just have Lewis Glacier outlines. An inventory for MtK is available for 2016, but not submitted. We decided to submit the full East Africa inventory from 2021 in a few weeks. Is there a need to digitize Diamond Glacier for 1993/2004 for consistency?

**Comment by iamdonovan** [comment]
No, no - I digitized all of the glaciers in the 1993 and 2004 maps, though there's a slight shift in the 1993 outline for Diamond Glacier due to what appears to be an error in the original maps.
